presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1797,6 +1798,34 @@ private void testBucketedTable(Session session, HiveStorageFormat storageFormat,
         assertFalse(getQueryRunner().tableExists(session, tableName));
     }
 
+    /**
+     * Regression test for https://github.com/prestosql/presto/issues/5295
+     */
+    @Test
+    public void testBucketedTableWithTimestampColumn()
+    {
+        String tableName = "test_bucketed_table_with_timestamp_" + randomTableSuffix();
+
+        String createTable = "" +
+                "CREATE TABLE " + tableName + " (" +
+                "  bucket_key integer, " +
+                "  a_timestamp timestamp(3) " +
```

Comment:
maybe use non-default value for precision. Like 5

Reply:
currently not supported

Reply:
currently not supported in create table flow

Reply:
Oh, right.